File: DataStruct/Mathematics/fnin.py
#Soln 1
from math import log

x = int(input("Enter the number?\n> "))
count = 0
while(x!=0):
    count+=1
    x//=10
  #  print(x)
print(count)


#Soln 2
x = int(input("\nEnter the number?\n> "))
print(int(log(x,10)+1))
input()

File: DataStruct/Mathematics/fnin.cpp
#include<stdio.h>
#include<math.h>
#include<iostream>
using namespace std;

void soln1()
{
  long long int x,count=0;
  cout<<"Enter the number?\n> ";
  cin>>x;
  while(x!=0)
  {
    count++;
    x/=10;
  }
  cout<<count<<endl;
}

void soln2()
{
  long long int x,c;
  cout<<"Enter the number?\n> ";
  cin>>x;
  c = log10(x)+1;
  cout<<c<<endl;
}

int main()
{
  soln1();
  soln2();
}
